include/srf/node/edge_builder.hpp

```diff
@@ -89,7 +150,6 @@ template <typename SourceT, typename SinkT>
 void operator|(SourceProperties<SourceT>& source, SinkProperties<SinkT>& sink)
```

Comment:
I think the Clara team is using a ` >> ` operator for denote an edge.  Since we don't have a way of chaining pipes, using a different operator might be more reasonable.

This has nothing to do with the review of this PR, but we should open an issue if we want to make this change.

Reply:
Yeah, I just spotted it on review. I'll open an issue.